jvm/src/main/kotlin/org/ionproject/core/CoreApplication.kt

```diff
@@ -120,6 +122,9 @@ class CoreSerializationConfig : WebMvcConfigurer {
     @Autowired
     lateinit var cache: AccessControlCache
 
+    @Autowired
```

Comment:
Do we really need these `lateinit @Autowired fields`? Can't we just use plain constructor injection?